pgx/src/datum/mod.rs

```diff
@@ -44,6 +45,7 @@ pub use item_pointer_data::*;
 pub use json::*;
 pub use numeric::*;
 use once_cell::sync::Lazy;
+pub use range::*;
```

Comment:
I have begun to want to replace these kinds of reexports with a "prelude" module. You may omit this.